src/components/Book.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+const Book = ({ title, author, category }) => (
+  (
+    <ul>
+      <li>{title}</li>
+      <li>{author}</li>
+      <li>{category}</li>
+    </ul>
```

Comment:
* Kindly add a button for removing books Note ✍️ the button wouldn't remove the books yet, this is just a placeholder for now 👍 
![image](https://user-images.githubusercontent.com/52497246/135082716-10dfa367-8697-4edc-be13-66804155e52e.png)

Reply:
Oh!:smile: 
Done! Thank you, @Davidosky007